code/getSampleRateAcc.m

```diff
@@ -0,0 +1,20 @@
+function sratesout = getSampleRateAcc(srates)
+%%
+% Input: matrix of sample rates of each packet from RawDataAccel.Json 
+% Output: sample rate in Hz 
+%%
+
+if length(unique(srates)) > 1 
+    warning('you have non uniform sample rates in your data'); 
+end
+
+% Convert sample rate codes to Hz
+sratesout = srates; 
+sratesout(srates==0) = 64;
+sratesout(srates==1) = 32;
+sratesout(srates==2) = 16;
+sratesout(srates==3) = 8;
+sratesout(srates==4) = 4;
+sratesout(srates==255) = NaN;
+
+clear temp*; 
```

Comment:
Same issue with "clear temp*" as before. Could cause other people code to break.

Reply:
Resolved with https://github.com/openmind-consortium/UCSF-rcs-data-analysis/pull/3/commits/24b2a683456a94d5de82d847a7684725a6331956